app/models/datastreams/dams_component_datastream.rb

```diff
@@ -99,127 +99,128 @@ def insertSourceCapture( solr_doc, cid, fid, sourceCapture )
     end
   end
           
-  def to_solr (solr_doc = {})
- 	storedInt = Solrizer::Descriptor.new(:integer, :indexed, :stored)
-    singleString = Solrizer::Descriptor.new(:string, :indexed, :stored)
-    storedIntMulti = Solrizer::Descriptor.new(:integer, :indexed, :stored, :multivalued)
-    facetable = Solrizer::Descriptor.new(:string, :indexed, :multivalued)
+ #  def to_solr (solr_doc = {})
```

Comment:
rather than comment out the redundant #to_solr method, can you please delete it? If, for some reason, we need to revert we'll have the code in the commit history.